core/datainput.gms

```diff
@@ -21,6 +21,9 @@ pm_globalMeanTemperature(tall)              = 0;
 pm_globalMeanTemperatureZeroed1900(tall)    = 0;
 pm_temperatureImpulseResponseCO2(tall,tall) = 0;
 
+*OR* initialize regional tax that is only required in 48_carbonpriceRegi
+pm_taxCO2eqRegi(ttot,regi)                 = 0;
```

Comment:
why do you declare it here and not in 48_carbonpriceRegi?

Reply:
So you would prefer moving this to 48_carbonpriceRegi/none?

Reply:
yes